src/components/Footer/Footer.js

```diff
@@ -1,17 +1,44 @@
 import React from "react";
 import styles from "../Footer/Footer.module.css";
 import PropTypes from "prop-types";
+import classnames from "classnames";
 
-const Footer = ({ count }) => (
+const Footer = ({
+  count_all,
+  count_end,
+  count_inWork,
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Identifier 'count_inWork' is not in camel case.](https://app.codacy.com/gh/WebHeroSchool/tododo/pullRequest?prid=5504860)